src/extension.ts

```diff
@@ -19,10 +19,10 @@ export async function activate(context: vscode.ExtensionContext) {
   )
     return
 
-  if (
-    !getConfig().enable
-    && !(await isVitestEnv(vscode.workspace.workspaceFolders[0].uri.fsPath))
-  )
+  const vitestEnvironmentFolders = vscode.workspace.workspaceFolders.filter(async folder =>
```

Comment:
filter cannot take async function as param. async func will always return Promise, which will be taken as True in filter